cdk/lib/admin-console.ts

```diff
@@ -84,6 +84,31 @@ export class AdminConsole extends GuStack {
     return table;
   }
 
+  buildArchivedTestsTable(): Table {
+    const id = `ArchivedChannelTestsDynamoTable`;
+
+    const table = new Table(this, id, {
+      tableName: `support-admin-console-archived-channel-tests-${this.stage}`,
+      removalPolicy: RemovalPolicy.RETAIN,
+      pointInTimeRecovery: this.stage === 'PROD',
+      billingMode: BillingMode.PAY_PER_REQUEST,
+      partitionKey: {
+        name: 'channel',
+        type: AttributeType.STRING,
+      },
+      sortKey: {
+        name: 'name',
+        type: AttributeType.STRING,
+      },
+    });
+
+    // Give it a better name
+    const defaultChild = table.node.defaultChild as unknown as CfnElement;
```

Comment:
Out of interest, what doesn't TS like about this? What type does it think `defaultChild` has without the casting?

Reply:
It's an `IConstruct`, which `CfnElement` also extends

Reply:
Got it, thanks. But `IConstruct ` doesn't have `.overrideLogicalId`?